### PR TITLE
Add update PATH instructions

### PR DESCRIPTION
**Pull request by TylerWoolcott**
Added update PATH instructions after installing Solana before running ``solana --version`` in terminal to avoid ``zsh: command not found: solana``

### CODE REVIEW

**Comment by adilanchian** [comment]
Hey @TylerWoolcott !

Thanks for the change request here :). When you were installing this, did it not instruct you to add this export at some point 🤔

Usually it does!

**Comment by TylerWoolcott** [comment]
Hi @adilanchian, 

Yes there is a subtle mention to add the export in the terminal but I wasn't looking for it as it's not mentioned in the docs even though it won't work otherwise and it's not easy to see even once aware. In addition, as I was closely following the 'Setting up Solana on a M1 macOS Machine' docs and it isn't mentioned I didn't think to look for it / do it. Plus I just got finished troubleshooting the 'Having problems with greadlink' issue on my machine so wasn't expecting to get an error running ``solana --version`` and didn't know if the error was connected to my ``greedlink`` error or if it was a different error. It took me a long time to realise what was going on and want to save others who might encounter this the time and confusion.

**Comment by adilanchian** [comment]
Hey @TylerWoolcott !

Thanks for the info here :). Yup this is a good add for sure based on your troubleshooting. I appreciate the steps you took!

Please see my review comments :)

File: Solana_And_Web3/en/Section_2/Resources/m1_setup.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ rustup toolchain install stable
 
 </details>
 
-This might take some time, so don't be alarmed! Once you're done installing, run this to make sure everything is in working order:
+This might take some time, so don't be alarmed! Once you're done installing, you may need to ensure it is in your path by pasting in your terminal ``export PATH="/Users/<your_username_here>/dev/solana"/bin:"$PATH"`` and replacing ``<your_username_here>``, then run this to make sure everything is in working order:
```
Comment (adilanchian) [inline Solana_And_Web3/en/Section_2/Resources/m1_setup.md:109]: So in solana cargo repo, their log statement actually looks like this - 

`echo "export PATH=\"$installDir\"/bin:\"\$PATH\""`

I think we should change you `<your_username_here>` with `<install_path>` thoughts?

**Review by adilanchian — CHANGES_REQUESTED** [review]

File: Solana_And_Web3/en/Section_2/Resources/m1_setup.md (hunk 1 above)
Comment (TylerWoolcott) [reply Solana_And_Web3/en/Section_2/Resources/m1_setup.md:109]: Hi @adilanchian, 

Great and yes agreed ``<install_path>`` is more in keeping with their original log statement.

Comment (adilanchian) [reply Solana_And_Web3/en/Section_2/Resources/m1_setup.md]: Thanks for the change here!

Sorry - I should have been a little more clear here, I think it should actually look like this:

`PATH="< install_path >/dev/solana"/bin:"$PATH"`

Let me know your thoughts!! Just in the chance that someone actually installs this somewhere else you know?

Comment (adilanchian) [reply Solana_And_Web3/en/Section_2/Resources/m1_setup.md]: Also looks like there are a few conflicts ! If you need some help resolving those, lemme know!

**Comment by TylerWoolcott** [comment]
Thanks @adilanchian, 

Yes good point, I made those changes, resolved the conflict and pushed a new commit (hash # 05896d7)

**Comment by adilanchian** [comment]
Thanks for the changes here - really appreciate it!!

**Pull request merged**